runtime/vm/createramclass.cpp

```diff
@@ -2857,7 +2857,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 						J9ARRAYCLASS_SET_STRIDE(ramClass, J9_VALUETYPE_FLATTENED_SIZE(elementClass));
 					}
 				} else {
-					J9ARRAYCLASS_SET_STRIDE(ramClass, (1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
+					J9ARRAYCLASS_SET_STRIDE(ramClass, (((U_64) 1) << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```

Comment:
```suggestion
					J9ARRAYCLASS_SET_STRIDE(ramClass, (((UDATA) 1) << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```

Reply:
`SET_STRIDE` already casts to `UDATA`:

```#define J9ARRAYCLASS_SET_STRIDE(clazz, strideLength) ((clazz)->flattenedClassCache) = (J9FlattenedClassCache*)(UDATA)(strideLength)```

Reply:
The cast needs to occur before the shift, the one in the macro occurs after

Reply:
Ah, that makes sense. Will commit the change